components/salesforce_rest_api/salesforce_rest_api.app.mjs

```diff
@@ -326,6 +326,15 @@ export default {
         data,
       });
     },
+    async createUser({ $, data }) {
+      const url = this._sObjectTypeApiUrl("User");
+      return this._makeRequest({
+        $,
+        url,
+        method: "POST",
+        data,
```

Comment:
```suggestion
    async createUser(opts = {}) {
      const url = this._sObjectTypeApiUrl("User");
      return this._makeRequest({
        url,
        method: "POST",
        ...opts,
```

Reply:
This suggestion is inconsistent with the larger component.

Reply:
Sorry, what is the inconsistency?

Reply:
async createAccount({
      $, data,
    }) {
      const url = this._sObjectTypeApiUrl("Account");
      return this._makeRequest({
        $,
        url,
        method: "POST",
        data,
      });
    },
    async updateAccount({
      $, id, data,
    }) {
      const url = this._sObjectDetailsApiUrl("Account", id);
      return this._makeRequest({
        $,
        url,
        method: "PATCH",
        data,
      });
    },
    async createAttachment({
      $, data,
    }) {
      const url = this._sObjectTypeApiUrl("Attachment");
      return this._makeRequest({
        $,
        url,
        method: "POST",
        data,
      });
    },
    async createUser({ $, data }) {
      const url = this._sObjectTypeApiUrl("User");
      return this._makeRequest({
        $,
        url,
        method: "POST",
        data,
      });
    },
    async createCampaign({
      $, data,
    }) {
      const url = this._sObjectTypeApiUrl("Campaign");
      return this._makeRequest({
        $,
        url,
        method: "POST",
        data,
      });
    },
    async createCase({
      $, data,
    }) {
      const url = this._sObjectTypeApiUrl("Case");
      return this._makeRequest({
        $,
        url,
        method: "POST",
        data,
      });
    },
    async createCaseComment({
      $, data,
    }) {
      const url = this._sObjectTypeApiUrl("CaseComment");
      return this._makeRequest({
        $,
        url,
        method: "POST",
        data,
      });
    },
    async createContact({
      $, data,
    }) {
      const url = this._sObjectTypeApiUrl("Contact");
      return this._makeRequest({
        $,
        url,
        method: "POST",
        data,
      });
    },
    async updateContact({
      $, id, data,
    }) {
      const url = this._sObjectDetailsApiUrl("Contact", id);
      return this._makeRequest({
        $,
        url,
        method: "PATCH",
        data,
      });
    },
    async createEvent({
      $, data,
    }) {
      const url = this._sObjectTypeApiUrl("Event");
      return this._makeRequest({
        $,
        url,
        method: "POST",
        data,
      });
    },
    async createLead({
      $, data,
    }) {
      const url = this._sObjectTypeApiUrl("Lead");
      return this._makeRequest({
        $,
        url,
        method: "POST",
        data,
      });
    },
    async createNote({
      $, data,
    }) {
      const url = this._sObjectTypeApiUrl("Note");
      return this._makeRequest({
        $,
        url,
        method: "POST",
        data,
      });
    },
    async createOpportunity({
      $, data,
    }) {
      const url = this._sObjectTypeApiUrl("Opportunity");
      return this._makeRequest({
        $,
        url,
        method: "POST",
        data,
      });
    },
    async updateOpportunity({
      $, id, data,
    }) {
      const url = this._sObjectDetailsApiUrl("Opportunity", id);
      return this._makeRequest({
        $,
        url,
        method: "PATCH",
        data,
      });
    },
    async getRecordFieldValues(sobjectName, {
      $, id, params,
    }) {
      const url = this._sObjectDetailsApiUrl(sobjectName, id);
      return this._makeRequest({
        $,
        url,
        params,
      });
    },
    async createRecord(sobjectName, {
      $, data,
    }) {
      const url = this._sObjectTypeApiUrl(sobjectName);
      return this._makeRequest({
        $,
        url,
        method: "POST",
        data,
      });
    },
    async updateRecord(sobjectName, {
      $, id, data,
    }) {
      const url = this._sObjectDetailsApiUrl(sobjectName, id);
      return this._makeRequest({
        $,
        url,
        method: "PATCH",
        data,
      });
    },
    async createTask({
      $, data,
    }) {
      const url = this._sObjectTypeApiUrl("Task");
      return this._makeRequest({
        $,
        url,
        method: "POST",
        data,
      });
    },
    async deleteOpportunity({
      $, id,
    }) {
      const url = this._sObjectDetailsApiUrl("Opportunity", id);
      return this._makeRequest({
        $,
        url,
        method: "DELETE",
      });
    },

Reply:
It was just a suggestion, feel free to update the others if you want, or keep your code this way. It doesn't affect the behavior of the code.